README.md

```diff
@@ -1,2 +1,58 @@
+
+Consistent engagement in programming will help partcipants build a solid foundation in programming, enabling them to take on more complex challenges. Participants will also develop a portfolio of skills and work experience for the world to see!
+
+### DevOps  
+
+This GSD-IT track focuses heavily on devops concepts and best practices using AWS. Docker, Jenkins, Terraform, and Amazon Web Service (AWS) are just some of the products and technologies used within this track.
+
+***DevOps: General Tasks***  
+
+Participants will apply fundamental concepts and utilize essentials tools such as Jenkins, Docker, and AWS. The will learn to use and create IaaS, PaaS, and SaaS products and services. Participants will explore concepts such as Infrastructure as Code (IaC) & Continous Integration / Continuous Deployment (CI/CD) to complete tasks for inDevelopme's open-source libraries. A participant may be asked to learn about containerizaiton or container orchestration tools -- and then be asked to demonstration competency through an implementation. This has a heavy focus in sucrity best practices, collaboration, application architecture, and documentation. 
+  
+***DevOps: Benefits to Participant***  
+
+Consistent engagement in the devops track will help partcipants build a solid foundation in cloud architecture. Participants will develop skill that help solve high availability, reliability, and scalability issues within an cloud application. 
+
+### Content Strategy
+
+This GSD-IT track focuses heavily content development best practices in the digital space. This track uses technologies and concepts such as Squarespace, Stripe, Medium, Google Analytics, etc to teach participants IT concepts around website development. This is done to help participants in the Business Administration and Fine Arts field better understand the role that IT plays in the content development life cycle.
+  
+***Content Strategy: General Tasks***  
+
+Participants on the Content Strategy track will complete tasks related to the content development for inDevelopme's non-profit initiatives. This will include the development of style guides, voice and tone, user personas, etc for inDevelopme as outlined by inDevelopme's Content Playbook. Participants will also utilize Search Engine Optimization (SEO) and Marketing (SEM) best practices to raise content engagement for JustAskDavidB and other inDevelopme non-profit initiatives.
+
+***Content Strategy: Benefit to Participant***  
+
+Consistent engagement in this track will establish solid foundation in the content developemnt lifecycle and general IT concepts such "development environments" and User Acceptance Testing (UAT). Participants will occassional have working sessions with content strategist that have 15+ years of experience in the field. These opportunities will help participants become proficient at making strategic decisions backed by research and data for the purpose of increasing content engagement.  
```

Comment:
- insert "a" before "solid foundation"
- "life cycle" is two words
- why is development environments in quotations but not UAT?
- change "occasional" to "occasionally" 
- make "strategist" plural